install

```diff
@@ -277,7 +277,7 @@ if [[ -f $INSTALL_DIR/etc/cron.weekly/fstrim ]]; then
 fi
 
 echo "==> Disabling PasswordAuthentication"
-sed s/PasswordAuthentication\ yes/PasswordAuthentication\ no/ -i $INSTALL_DIR/etc/ssh/sshd_config
+sed -ri s/^#?PasswordAuthentication\ yes/PasswordAuthentication\ no/ $INSTALL_DIR/etc/ssh/sshd_config
```

Comment:
This work work as expected since the default is typically:

```
#PasswordAuthentication no
```

So I think it would be better to also include:

```
sed -ri s/^#?PasswordAuthentication\ no/PasswordAuthentication\ no/ $INSTALL_DIR/etc/ssh/sshd_config
```